pyproject.toml

```diff
@@ -11,14 +11,15 @@ authors = [
 python = "^3.11"
 django = "^5.1"
 whitenoise = "^6.8.2"
-drunc = { git = "https://github.com/DUNE-DAQ/drunc.git" }
-druncschema = { git = "https://github.com/DUNE-DAQ/druncschema.git" }
+drunc = { git = "https://github.com/DUNE-DAQ/drunc.git", rev = "f068eaccb840da03a104adba2da3f27454c218d5" }
+druncschema = { git = "https://github.com/DUNE-DAQ/druncschema.git", rev = "a07226fa9c3a4329b4c38e084c913cb1e2db0b4e" }
```

Comment:
This means it relies on a fixed version of drunc and druncschema and won't pick up on any future updates. Is this intentional so that it matches the nightly build we have and updated versions will only be brought in after they're tested alongside this?

Reply:
Yeah exactly. We want the version the app is using to exactly match the development environment. It's nightly build so that's just some random commit. Hopefully we'll be able to move to a new release at some point.